readme.txt

```diff
@@ -2,7 +2,7 @@
 Contributors: joemcgill, pacifika, kburgoine, tweetythierry, swissspidy
 Tags: seo, sitemaps
 Requires at least: 5.3
```

Comment:
Let's also bump this to 5.4 while we're at it

Reply:
👍

Reply:
Bumped in https://github.com/GoogleChromeLabs/wp-sitemaps/pull/173/commits/5fadaecb87c0822660e05241f3eaf65e6a169824